### PR TITLE
An iteration of the BinderHub and JupyterHub ecosystem entries

### PR DESCRIPTION
**Pull request by consideRatio**
This is a followup on @lheagy's work with some lesser changes to the BinderHub and JupyterHub documentation.

### Changes
- Converted .ipynb files to markdown for my own convinience and ease of version control
- General pass of the text overall retaining it as it were but attempting to make it even more approachable.

### Preview
- [JupyterHub section](https://github.com/pangeo-data/jupyter-earth/blob/b1cbcbd93ea1247109691e53aac461c18d7ab88b/docs/jupyter-resources/ecosystem/jupyterhub.md)
- [BinderHub section](https://github.com/pangeo-data/jupyter-earth/blob/b1cbcbd93ea1247109691e53aac461c18d7ab88b/docs/jupyter-resources/ecosystem/binder.md)

### CODE REVIEW

File: docs/jupyter-resources/ecosystem/binder.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+## Terminology
+
+As _Binder_ is often casually used to refer to multiple things, it's useful to distinguish that:
+- _mybinder.org_ is a free and public service.
+- _[BinderHub](https://binderhub.readthedocs.io/en/latest/)_ is the open source software used by mybinder.org.
+- A BinderHub-ready git repository could be referred to as _a binder_.
+
+## Creating a BinderHub-ready git repository
+
+To make your notebooks accessible with mybinder.org, you have to make them public in a git repository and declare whats required for them to run properly.
```
Comment (andersy005) [inline docs/jupyter-resources/ecosystem/binder.md:16]:
```suggestion
To make your notebooks accessible with mybinder.org, you have to make them public in a git repository and declare what is required for them to run properly.
```

File: docs/jupyter-resources/ecosystem/binder.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+### Recommended guides
+
+While we provide some details below, we highly recommend the following more detailed external and programming language tailored guides:
```
Comment (andersy005) [inline docs/jupyter-resources/ecosystem/binder.md:20]:
```suggestion
While we provide some details below, we highly recommend the following external, more detailed programming language tailored guides:
```

**Review by andersy005 — COMMENTED** [review]
@consideRatio, I left a few inline comments/suggestions.

**Comment by consideRatio** [comment]
Thanks @andersy005!

**Review by whyjz — COMMENTED** [review]
Thanks @consideRatio! The changes look great to me. Just one quick thought: it would be great to change `docs/_toc.yml` to let JupyterBook source these md files instead of the removed notebook files.

**Comment by consideRatio** [comment]
@whyjz ah that was a very relevant point! =D I removed the file extensions from the _toc.yml file as jupyter-book will automatically detect it if there is only one file with that name, which was what I assumed.

**Comment by consideRatio** [comment]
Thank you @andersy005 and @whyjz! :heart: :tada:!

**Pull request merged**